build.gradle

```diff
@@ -157,7 +160,9 @@ dependencies {
     compile "org.json:json:${jsonVer}"
     compile "com.google.inject:guice:${guiceVer}"
     compile "com.google.inject.extensions:guice-assistedinject:${guiceVer}"
-    compile "org.glassfish.jersey.containers:jersey-container-jetty-http:2.23.2"
+    compile "org.glassfish.jersey.containers:jersey-container-jetty-http:${jerseyContainerVer}"
+    compile "org.eclipse.jetty:jetty-servlet:${jettyServletVer}"
+    compile "com.owlike:genson:${gensonVer}"
```

Comment:
The advantage was that it worked, when others did not.  I can hunt around for something that works from someone else.